downstream/assemblies/platform/assembly-set-up-automation-mesh.adoc

```diff
@@ -0,0 +1,25 @@
+
+ifdef::context[:parent-context: {context}]
+
+
+[id="set-up-automation-mesh"]
```

Comment:
```suggestion
[id="setting-up"]
```

Reply:
I would personally also change this in the file name. `s/assembly-set-up-automation-mesh.adoc/assembly-setting-up-automation-mesh.adoc`

One note here is that I think it's alright to use product names in the file names. That doesn't get exposed in the URL and can be helpful when you're digging through the filesystem.